README.md

```diff
@@ -41,44 +21,118 @@ This allows you to check in any changes you make to your private copy of the sol
     |- pipeline               - automated testing pipeline source files
 ```
 
-## Deploy the Solution From Github
+# Deploying the Solution
+***
+## One-Click Deploy From Amazon Web Services
+Refer to the [solution landing page](https://aws.amazon.com/solutions/implementations/instance-scheduler-on-aws/)
+to deploy Instance Scheduler on AWS using our pre-packaged deployment assets
+
+## Deploy from GitHub (CDK Deploy)
+
+Instance Scheduler can be deployed your AWS account directly from the source code using `cdk deploy`
+
+### Deploying the Primary Control Stack
+```
+cd source/infrastructure
+npm install
+cd aws-instance-scheduler
+cdk bootstrap
+cdk deploy aws-instance-scheduler
+```
+
+This will deploy the solution into your aws account using all default configuration settings. 
+You can then update those settings to their desired values from the CloudFormation console 
+by selecting the deployed template and clicking "Update" -> "Use Current Template"
+
+Refer to the [Implementation Guide](https://docs.aws.amazon.com/solutions/latest/instance-scheduler-on-aws/deployment.html#step1)
+for guidance on what each of the configuration parameters is for
+
+### Deploying Remote Stacks in Other Accounts
+To deploy the remote stack for cross-account scheduling, you will first need to have deployed the primary
+control stack. Then update your aws credentials to match those of the remote account you would like to schedule
+and  deploying the remote stack
+```
+//source/infrastructure/aws-instance-scheduler
+
+cdk deploy aws-instance-scheduler-remote --parameters InstanceSchedulerAccount={account-id}
+```
+replace {account-id} with the id of the account that contains the primary control stack.
+
+For example: `InstanceSchedulerAccount=111222333444`
+
+
+## Deploy from GitHub (AWS Console)
+This method mimics the procedure used by AWS One-Click Deploy allowing you to deploy the solution
+from the AWS console using assets that you can control and update. 
 
+### Overview
 AWS Solutions use two buckets: a bucket for global access to templates, which is accessed via HTTPS, and regional buckets for access to assets within the region, such as Lambda code. You will need:
 
-* One global bucket that is access via the http end point. AWS CloudFormation templates are stored here. Ex. "mybucket"
+* One global bucket that is accessed via the https end point. AWS CloudFormation templates are stored here. Ex. "mybucket"
 * One regional bucket for each region where you plan to deploy using the name of the global bucket as the root, and suffixed with the region name. Ex. "mybucket-us-east-1"
 * Your buckets should be encrypted and disallow public access
 
-**Build the solution**
+#### You will need:
+
+* a Linux client with the AWS CLI v2, python 3.7+, NPM, and Docker
+* a clone (or fork) of this repo
+* two S3 buckets (minimum): 1 global and 1 for each region where you will deploy
```

Comment:
Do you really need a linux client? Don't we need a newer python version than 3.7? Specific node and npm versions? Docker meaning docker desktop or docker engine?

Reply:
the build script is a shell script that requires a linux-compatible terminal (as I know all too well >_>). the rest of it I'm not sure how to verify the actual versions which are needed but wanted to include some mention of them at least.

Reply:
If you're not sure, I wouldn't use specifics here. You don't have to specify the bare-minimum requirements, you can impose restrictions like "you should have the python3.9 because that's what the lambda runs on".

Reply:
3.7+ is what was in the file originally. Not sure where it came from but I think 3.9+ makes sense. alternatively I could just say "latest versions of" for both python and node